scripts/gsp-local.sh

```diff
@@ -87,12 +87,15 @@ function apply() {
 }
 
 log "Creating local GSP..."
+minikube start \
+	--memory 8192 \
+	--cpus 4 \
+	--disk-size 30g \
+	--vm-driver hyperkit \
+	--kubernetes-version v1.14.0 \
```

Comment:
Should we make this the same version as the eks cluster?

Reply:
Probably, I'll tweak it now.